fundamentals/arrays/lesson_notes/lesson_01.md

```diff
@@ -0,0 +1,137 @@
+# Arrays, Standard Operations
+
+## Setup
+
+Please complete the following before the lesson begins:
+
+- Fork this gist. Then, if you wish to take notes in the gist itself, click the “Edit” button.
+- Open and then fork this [Replit](https://replit.com/@bwreid/Arrays-Part-I).
+- Complete the Warmup Questions & Tasks detailed at the top of the Replit.
+
+## Learning Objectives
+
+By the end of this lesson you will be able to:
+
+- Describe the purpose of arrays and their unique properties.
+- Create new array literals with open and closing brackets.
+- Access elements of an array at a given index.
+- Access the length of an array.
+- Use array methods to remove elements from the front and back of an array.
+- Use array methods to add elements to the front and back of an array.
+- Assign values in an array at a particular index with bracket notation.
+
+## Guiding Questions
+
+- What can arrays do that strings, numbers, booleans, undefined, and null cannot do?
+
+  > Your answer…
+
+- Arrays are collections of elements. What kinds of data types can be elements within in array?
+
+  > Your answer…
+
+- Take a look at the `teaOptions` array below. What characters let you know that this is an array?
+
+  ```js
+  let teaOptions = ["Earl Grey", "Jasmine", "Mint"];
+  ```
+
+  > Your answer…
+
+- How can you determine the length of an array without manually counting all of the elements inside of the array?
+
+  > Your answer…
+
+- At the bottom of your Repl.it, copy and paste the following code. Before pressing the “Run” button, consider what you think will happen. Then, press “Run” to confirm your hypothesis. Did you guess right?
+
+  ```js
+  console.log([hobby1, age1, hobby2]);
+  ```
+
+- In your Repl.it, create a new variable with a name of `myArray`. Then, assign it the value of an array that includes five elements. You may use the existing variables to populate the array.
+
+  If you wanted to access the first item in the array, how would you do so?
```

Comment:
In the interest of time, I think having the syntax in this markdown of accessing the first element would be helpful, and then ask them about the second/third item.

Reply:
Let's discuss this. The overall intention is that students:
1. Read the Readme associated with this content.
2. Then participate in the lesson with these notes.

Recalling information you've seen previously, or attempting to discover the information on your own, has shown to increase learning retention rates and is generally better than giving students the answer.

Does having the Readme as a resource during this lesson not provide enough guidance?